code/gateway/daqrest.q

```diff
@@ -0,0 +1,44 @@
+.aqrest.execute:{[req;props] @[value;req;{(neg .z.w)(.gw.formatresponse[0b;0b;"error: ",x])}]};
+
+\d .dataaccess
+
+// .gw.formatresponse:{[status;sync;result]$[not[status]and sync;'result;result]}};
```

Comment:
Is this a comment or a commented out line of code?

Reply:
It's the default definition of the function would you rather it deleted?

Reply:
If its needed keep it but make it clear its a comment versus a commented line of code